src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -52,58 +52,64 @@
     @ResponseStatus(HttpStatus.FORBIDDEN)
```

Comment:
why you did not add localed messages for custom exceptions
 MessageDTO(emptyFileException.getMessage());

Reply:
It's not a purpose of this taks.
I didn't add any new messages, just moved them to resource file.

Reply:
you haven't to add new message
you just need move it  from exceptions class (from body of exeption) to your message r.
it's nessesary